lib/cretonne/src/ir/dfg.rs

```diff
@@ -106,6 +106,23 @@ impl DataFlowGraph {
     }
 }
 
+/// Resolve value aliases.
+///
+/// Find the original SSA value that `value` aliases.
+fn resolve_aliases(values: &EntityMap<Value, ValueData>, value: Value) -> Value {
```

Comment:
I wonder if we would get better code if this function takes a `value: &mut Value` argument instead of returning the new value?

When inlined into `resolve_aliases_in_arguments`, there would be one less test.